### PR TITLE
duplicate links are references in markdown 

### PR DESCRIPTION
**Pull request by Romicodes**
@muffinresearch @caitmuenster  I was able to achieve a reasonable measure of success with this bug #241 over the weekend and  have changed quite a few of the duplicate links into references in markdown. If you review  and let me know it is as expected, I can have everything updated by end of day 👍

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
+    activesupport (5.2.4.2)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
     addressable (2.7.0)
       public_suffix (>= 2.0.2, < 5.0)
     colorator (1.1.0)
-    concurrent-ruby (1.1.5)
+    concurrent-ruby (1.1.6)
```
Comment (championshuttler) [inline Gemfile.lock:12]: I do not think these are intended changes

File: content-guidelines/extension-subtitle.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ skip_index: true
 
 # Extension Subtitle
 
-The subtitle is one of your extension’s most important pieces of content. It’s the thing that shows up in search results, drawing a potential user’s attention. It makes your extension more discoverable. And, it’s used everywhere—in search results, throughout [addons.mozilla.org](https://addons.mozilla.org 'addons.mozilla.org'), and in the Firefox browser.
+The subtitle is one of your extension’s most important pieces of content. It’s the thing that shows up in search results, drawing a potential user’s attention. It makes your extension more discoverable. And, it’s used everywhere—in search results, throughout [addons.mozilla.org][https://addons.mozilla.org 'addons.mozilla.org'], and in the Firefox browser.
```
Comment (championshuttler) [inline content-guidelines/extension-subtitle.md:11]: This will disable the linking if you use `[]`

File: content-guidelines/extension-title.md [hunk 3]
```diff
@@ -8,9 +8,9 @@ skip_index: true
 
 # Extension Name
 
-Your extension’s name is critical. It shows up everywhere—in search results, on [addons.mozilla.org](https://addons.mozilla.org 'addons.mozilla.org'), and in the browser itself. It’s sometimes one of the few pieces of information that a user has to determine whether or not they are interested in what you built.
+Your extension’s name is critical. It shows up everywhere—in search results, on [addons.mozilla.org][https://addons.mozilla.org 'addons.mozilla.org'], and in the browser itself. It’s sometimes one of the few pieces of information that a user has to determine whether or not they are interested in what you built.
 
-Research shows that people really do take an extension’s name into account when making installation decisions. And, what you put into extension name field becomes your URL on [addons.mozilla.org](https://addons.mozilla.org 'addons.mozilla.org').
+Research shows that people really do take an extension’s name into account when making installation decisions. And, what you put into extension name field becomes your URL on [addons.mozilla.org][https://addons.mozilla.org 'addons.mozilla.org'].
```
Comment (championshuttler) [inline content-guidelines/extension-title.md:13]: Same here and every other places

File: content-guidelines/extension-subtitle.md (hunk 2 above)
Comment (Romicodes) [reply content-guidelines/extension-subtitle.md:11]: @championshuttler can the () be enclosed inside the []? Will that preserve the links?

Comment (championshuttler) [reply content-guidelines/extension-subtitle.md:11]: @Romicodes I don't think so https://guides.github.com/features/mastering-markdown/

Comment (Romicodes) [reply content-guidelines/extension-subtitle.md:11]: @championshuttler adding  [addons.mozilla.org] : https://addons.mozilla.org at the very end of the markdown page?

File: content-guidelines/extension-title.md (hunk 3 above)
Comment (Romicodes) [reply content-guidelines/extension-title.md:13]: @championshuttler  creating a key like
 [addons.mozilla.org] : https://addons.mozilla.org 'addons.mozilla.org'
 OR 
 [duplicate link]: https://addons.mozilla.org 'addons.mozilla.org' towards the end of  markdown file  be a good resolution?

**Comment by caitmuenster** [comment]
Hi @Romicodes! 👋🏼 I know it's been awhile but I wanted to check in and see if you would be interested in finishing off this PR?

**Comment by muffinresearch** [comment]
@Romicodes I'm going to close this as we haven't heard back from you, but if you'd like to continue with this feel free to re-open it and we can help you get this landed.

**Pull request closed without merging**